doc/source/tempfile.rst

```diff
@@ -76,6 +76,18 @@ You are allowed to specify the plasma object store socket in one of these ways:
 
 The path you specified will be given as it is without being affected any other paths.
 
+
+Raylet Socket
+--------------------------
+
+Raylet sockets can also be controlled by the user in one of these ways:
```

Comment:
I think it comes from https://github.com/ray-project/ray/pull/2862#issuecomment-425020680. Since we have exposed the plasma store, we can also expose raylet interface. Exposing raylet interface could also help https://github.com/ray-project/ray/issues/2594

Reply:
The plasma store is potentially different. I can imagine a use case where someone has their own plasma store and wants to use it across applications. I don't think that's true of the raylet.

Reply:
I think that adding documentation explaining how to do things that people will never want to do just makes it more confusing.

Reply:
OK. I remove that part.